### DISCUSSION TITLE
How many baur tasks do you have?

### DISCUSSION THREAD

**Discussion by kostyay** [simplesurance/baur#342] [Show and tell]
We already have near 1000 tasks :)

`Running 19/936 task(s)`

**Comment by fho** [comment]
Oh wow, we can't compete with that.  :-) We currently _only_ have 107 tasks in our monorepo.
I still didn't find time to really make use of the tasks, most of our apps only have a build task. I would like to create separate tasks for static checks, unit-tests,  etc.

Do you use a similar structuring? Or what kind of tasks do you have defined?

**Reply by kostyay** [reply]
We also have a single build task at the moment.
